src/gen/conditions.rs

```diff
@@ -4311,6 +4312,7 @@ fn test_eligible_for_ff_even_amount() {
 #[cfg(test)]
 #[rstest]
 #[case(123, "{h2}", true)]
+#[case(121, "{h2}", true)]
```

Comment:
the same puzzle-hash a different amount (but still odd) makes for an eligible FF spend